packages/compass-e2e-tests/tests/collection-aggregations-tab.test.ts

```diff
@@ -737,6 +705,11 @@ describe('Collection aggregations tab', function () {
   });
 
   it('shows the explain for a pipeline', async function () {
+    // Set first stage to $match
+    await browser.focusStageOperator(0);
+    await browser.selectStageOperator(0, '$match');
+    await browser.setAceValue(Selectors.stageEditor(0), '{ i: 5 }');
```

Comment:
We consider pipeline stage without operator or value broken and don't allow to run / export / explain aggregation in this state anymore